src/breadcrumb-group/item/item.tsx

```diff
@@ -112,17 +114,14 @@ const BreadcrumbItemWithPopover = <T extends BreadcrumbGroupProps.Item>({
 };
 
 type ItemProps = React.HTMLAttributes<HTMLElement> & {
-  dataAttributes?: React.DataHTMLAttributes<HTMLElement>;
   anchorAttributes: React.AnchorHTMLAttributes<HTMLAnchorElement>;
   isLast: boolean;
 };
-const Item = ({ anchorAttributes, dataAttributes, children, isLast, ...itemAttributes }: ItemProps) =>
+const Item = ({ anchorAttributes, children, isLast, ...itemAttributes }: ItemProps) =>
   isLast ? (
-    <span {...itemAttributes} {...dataAttributes}>
-      {children}
-    </span>
+    <span {...itemAttributes}>{children}</span>
   ) : (
-    <a {...itemAttributes} {...anchorAttributes} {...dataAttributes}>
+    <a {...itemAttributes} {...anchorAttributes}>
```

Comment:
The removed dataAttributes here were initially added by me for funnels but they were never used since we only add the attribute to the last item. Not sure why I added it, so removed it.